LayoutTests/imported/w3c/web-platform-tests/cookie-store/change_eventhandler_for_http_cookie_and_set_cookie_headers.https.window-expected.txt

```diff
@@ -1,9 +1,7 @@
 
-Harness Error (TIMEOUT), message = null
-
-TIMEOUT HTTP set/overwrite/delete observed in CookieStore Test timed out
-NOTRUN CookieStore agreed with HTTP headers agree on encoding non-ASCII cookies
-NOTRUN CookieStore set/overwrite/delete observed in HTTP headers
-NOTRUN HTTP headers agreed with CookieStore on encoding non-ASCII cookies
-NOTRUN Binary HTTP set/overwrite/delete observed in CookieStore
+FAIL HTTP set/overwrite/delete observed in CookieStore assert_equals: Deletion observed after HTTP cookie-clearing using max-age=0: number of changed cookies expected 0 but got 1
+FAIL CookieStore agreed with HTTP headers agree on encoding non-ASCII cookies assert_equals: Cookie we wrote using HTTP in cookie jar expected "HTTP-🍪=🔵" but got "HTTP-ðª=ðµ"
+FAIL CookieStore set/overwrite/delete observed in HTTP headers assert_equals: Observed cookie that was deleted: number of changed cookies expected 0 but got 1
+FAIL HTTP headers agreed with CookieStore on encoding non-ASCII cookies assert_equals: HTTP cookie jar contains only cookie we set expected "🍪=🔵" but got ""
+FAIL Binary HTTP set/overwrite/delete observed in CookieStore assert_equals: Binary cookie we wrote using HTTP in cookie jar expected "HTTP-cookie=value\ufffd" but got "HTTP-cookie=valueï¿½"
```

Comment:
I'm still looking into these new failures and the new failures in the other tests. Not yet 100% sure why they're happening.

Reply:
Could this be a case of a test simply getting further than it was before?

Reply:
It is probably fine to investigate this in a follow-up.